deploy.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Deploy docs.ongr.io
+  hosts: all
+  remote_user: dev
+  vars:
+    ansistrano_deploy_from: "./"
+    ansistrano_deploy_to: "/var/www/docs"
+    ansistrano_shared_paths: [app/logs, app/config, web/images]
```

Comment:
App config should not be shared. Only `app/config/parameters.yml` file. Other files should come from repository in that directory.

`web/images` folder are not necessary.